multi-agent/lib/mas/actions/providers/mcp/validate_connection/validate_connection.py

```
@@ -28,227 +28,88 @@
+                server_identifier=mcp_url,
                 response_time_ms=elapsed,
             )
-
         except TimeoutError:
             return ValidateConnectionOutput(
                 success=False, message="Connection timeout",
                 is_reachable=False,
                 response_time_ms=(time.time() - start) * 1000,
             )
-
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)